assets/js/translator.js

```diff
@@ -634,7 +634,16 @@ function populateTranslationList() {
     });
 
     $('#srcLangSelect').val(curSrcLang);
+    var currentSrcLangSelect = $('#srcLangSelect option[value="' + $('#srcLangSelect').val() + '"]');
+    if(currentSrcLangSelect.text().charCodeAt(0) === 160) {
+        currentSrcLangSelect.text(currentSrcLangSelect.text().substring(4)); /* sass-lint:disable-line no-magic-numbers */
```

Comment:
This isn't robust. There's no reason a variant can't be en_GB.

Reply:
@sushain97 I do not understand exactly what you mean by your statement. I tried adding in comments to see if it might clarify anything.

Reply:
Also, why isn't `/* sass-lint:disable-line no-magic-numbers */` working?

Reply:
> @sushain97 I do not understand exactly what you mean by your statement. I tried adding in comments to see if it might clarify anything.

Why does `4` work?

> Also, why isn't /* sass-lint:disable-line no-magic-numbers */ working?

 Why would it work?

Reply:
@sushain97 `4` works because there are 4 `&nbsp;`: `'&nbsp;&nbsp;&nbsp;&nbsp;'`.

Reply:
AH. My bad. Yeah, that should be commented.

Reply:
@sushain97 I did, see the new changes :D